lab-doug/lib/showContents.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var fs = require('fs');
+
+function showContents(){
+  fs.readdir(__dirname + '/../docs/', function(err, files){
```

Comment:
the name of the directory to read should probably be passed in to this method, not hardcoded.